tests/models/test_provider.py

```diff
@@ -125,7 +125,7 @@ def test_failing_regex_value() -> None:
     with pytest.raises(ValidationError, match="must match regex"):
         _ = VariantFeatureConfig(name="name", values=[""])
 
-    for c in "@#$%&*^()[]?!-{}[]\\/ ":
```

Comment:
Why did you remove that ? Maybe remove `!` which is now allowed. And you can add `\t`